hack/builder/Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM fedora:30
+ARG ARCH
+FROM ${ARCH}/fedora:30
```

Comment:
is there a technical reason to use docker here? is it possible to use centos 8 instead so that we'll not have to worry about image being removed from dockerhub?

Reply:
are there plans from the fedora people to not push images to dockerhub anymore?

Reply:
no, but since a new fedora version comes out every 6 month, and they don't keep all the versions on dockerhub, we will have to update the image here every now and then... not too often, but with centos we won't have to update for several years...

Reply:
True. We can discuss that. But maybe not in this PR?